src/Promitor.Agents.ResourceDiscovery/Graph/Authentication.cs

```diff
@@ -1,20 +1,55 @@
 ﻿using System.Threading.Tasks;
-using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Azure.Core;
+using Azure.Identity;
 using Microsoft.Rest;
+using Promitor.Agents.Core.Configuration.Authentication;
 
 namespace Promitor.Agents.ResourceDiscovery.Graph
 {
     public static class Authentication
     {
-        public static async Task<ServiceClientCredentials> GetServiceClientCredentialsAsync(string resource, string clientId, string clientSecret, string tenantId)
+        //public static async Task<ServiceClientCredentials> GetServiceClientCredentialsAsync(string resource, string clientId, string clientSecret, string tenantId)
```

Comment:
If you no longer need this, please remove it

Reply:
Oh, I just wanted to be sure nothing were lost (just in case) but obviously, this is not used anymore

Reply:
Kill it with 🔥 Thanks though :D